sentence_test.go

```diff
@@ -53,11 +53,6 @@ var sentencetests = []struct {
 		raw:  "%GPFOO,1,2,3,x,y,z*1A",
 		err:  "nmea: sentence does not start with a '$' or '!'",
 	},
-	{
-		name: "bad checksum delimiter",
-		raw:  "$GPFOO,1,2,3,x,y,z",
-		err:  "nmea: sentence does not contain checksum separator",
-	},
```

Comment:
Maybe instead of removing this, add the `*` delimiter without the actual checksum after.